src/Components/Navbar.js

```diff
@@ -6,25 +6,26 @@ function Navbar() {
     <StyledNavbar>
       <ul>
         <li>
-          <StyleNavLink to="/" end>
+          <StyleNavLink to="/notes" end>
             <svg
```

Comment:
Nice! Just a suggestion. You can put all your icons in a  separate component, for example, an `Icons` Component, then import them wherever they are needed. This way your code looks cleaner.